<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnAWindow</class>
 <widget class="QDialog" name="QnAWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bebas Neue</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Community Service | Help</string>
   </property>
  </widget>
  <widget class="QListWidget" name="helpList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Q: How to add a member?</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: Add a member by</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: Where can I find the source code?</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: In the home page, the action menu contains a button called &quot;Github&quot;. </string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>    This will open up the website containing the source code.</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: What if I loose my password [Member]?</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: Contact your administrator,  they have access to your password.</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: What if I loose my password [Admin]?</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: Inside the program file, delete &quot;adminLogin.txt&quot;. Next time you login</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>    you will be given temporary credentials that, can changed inside</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>    the admin panel</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: How can I edit a member's information?</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: In the admin panel, clicking the &quot;Edit&quot; button</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: How can I view/edit a member's events?</string>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: Once the member is chosen, click on &quot;View Events&quot;, this will provide</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>    you the ability to view and edit an event's information</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: How to export data to Excel or other similar mediums?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: In the admin panel, click on &quot;Export Data&quot; and choose a save location</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: How to backup my data?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: In the admin panel, click on &quot;Backup Data&quot;</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: Will retriving my backup data keep the new additions?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: No, retriving your backed up data will restore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>    to its original backup state</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Q: How many hours for the CSA Award?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A: A minum of 15 hours is required</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>531</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="font">
         <font>
          <family>Kenyan Coffee</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
